presto-main/src/main/java/io/prestosql/event/QueryMonitor.java

```diff
@@ -454,6 +456,9 @@ private static void logQueryTimeline(QueryInfo queryInfo)
             long executionInitializationTime = max(firstTaskStartTime - firstStageFirstTaskStartTime, 0);
             long running = max(lastTaskEndTime - firstTaskStartTime, 0);
             long finishing = max(queryEndTime.getMillis() - lastTaskEndTime, 0);
+            int nodes = globalUniqueNodes(outputStage).size();
+            long spilledWriteTimeMillisPerNode = (queryStats.getSpilledWriteTime().toMillis() / nodes);
```

Comment:
should nodes include the nodes where spill occurred?

Reply:
Fixed